cmd/kuberhealthy/kuberhealthy.go

```diff
@@ -1171,15 +1164,15 @@ func (k *Kuberhealthy) validateExternalRequest(remoteIPPort string) (PodReportIP
 		envVars = append(envVars, container.Env...)
 	}
 
-	log.Debugln("Env vars found on pod with IP", ip, envVars)
+	log.Debugln("Env vars found on pod with kh run uuid", uuid, envVars)
 
 	// validate that the environment variables we expect are in place based on the check's name and UUID
 	// compared to what is in the khcheck custom resource
 	var foundUUID bool
 	for _, e := range envVars {
 		log.Debugln("Checking environment variable on calling pod:", e.Name, e.Value)
 		if e.Name == external.KHRunUUID {
-			log.Debugln("Found value on calling pod", ip, "value:", external.KHRunUUID, e.Value)
+			log.Debugln("Found value on calling pod", uuid, "value:", external.KHRunUUID, e.Value)
```

Comment:
kept most of this logic in here -- sort of seems repetitive since the uuid set in the header is from the environment variable and we validate it again here, thoughts? but when you go further down, this uuid grabbed from the env var here is compared to its khstate current run UUID which I think is important @integrii